src/Microsoft.OpenApi/Services/OpenApiWalker.cs

```diff
@@ -603,12 +674,22 @@ internal void Walk(IOpenApiElement element)
                 case OpenApiServerVariable e: Walk(e); break;
                 case OpenApiTag e: Walk(e); break;
                 case IList<OpenApiTag> e: Walk(e); break;
-                case OpenApiXml e: Walk(e); break;
                 case IOpenApiExtensible e: Walk(e); break;
                 case IOpenApiExtension e: Walk(e); break;
-
             }
         }
 
+        /// <summary>
+        /// Adds a segment to the context path to enable pointing to the current locationin the document
+        /// </summary>
+        /// <param name="context">An identifier for the context.</param>
+        /// <param name="walk">An action that walks objects within the context.</param>
+        private void InContext(string context, Action walk)
```

Comment:
nit, InContext is a little strange as a method name. 

This is still a Walk method to me (just with a context name). Maybe it should be called "Walk" or "WalkInto"?